edx_proctoring/tests/test_api.py

```diff
@@ -14,8 +14,8 @@
     stop_exam_attempt,
     get_active_exams_for_user,
     get_exam_attempt,
-    create_exam_attempt
-)
+    create_exam_attempt,
+    get_student_view)
```

Comment:
Please use a trailing comma and put the close-paren on its own line, so that future diffs will be cleaner.